1-getting-started-lessons/2-github-basics/translations/README.pt.md

```diff
@@ -80,7 +80,7 @@ Digamos que você tenha uma pasta localmente com algum projeto de código e dese
         modified:   file2.txt
    ```
 
-   Geralmente o comando `git status` diz a você coisas como quais arquivos estão prontos para serem _salvos_ para o repo ou tem alterações que você pode querer persistir.
+   Geralmente o comando `git status` diz a você coisas como: quais arquivos estão prontos para serem _salvos_ para o repo ou tem alterações que você pode querer persistir.

```

Comment:
Is this colon needed here?

Reply:
Yes, it exemplifies what git can accomplish.

Reply:
@jlooper Would you like to remove the colon?

Reply:
Done!